src/functions-and-arrays.js

```diff
@@ -1,19 +1,119 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers (a, b) {
+  if (a > b) {
+    return a;
+  }
+  return b
+};
+
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+
+
+function findLongestWord(arr) {
+  
+
+  if (typeof arr == 'undefined' || arr.length < 1){
+    return null;
+  } 
+
+  let longestWord = 'a';
```

Comment:
I would rather have this to be: let longestWord = ''
To prevent any potential bugs